app/javascript/packs/navigations/validator_searcher_btn_template.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <button class="btn btn-lg btn-secondary"
+          id="validator-searcher-btn"
+          :class="searcher_class()"
+          v-on:click="trigger_btn()">
+    <i class="fas fa-search"></i>
+  </button>
+</template>
+
+<script>
+  export default {
+    data() {
+      return {
+        is_active: false
+      }
+    },
+    methods: {
+      searcher_class() {
+        return this.is_active ? 'active' : ''
+      },
+      trigger_btn() {
+        this.is_active = !this.is_active
+
+        const validator_searcher_bar = document.getElementById("validator-searcher-bar")
+
+        if (this.is_active) {
```

Comment:
how about delete ifs and do only 
```suggestion
        validator_searcher_bar.classList.toggle("active")
```

Reply:
It can work out. Thanks 👍